src/microbe_stage/BindingExtensions.cs

```diff
@@ -0,0 +1,215 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+
+public static class BindingExtensions
```

Comment:
Why is this a class?

Reply:
This is an extention class. This seperates colony logic from microbe logic.

Reply:
Making microbe a partitial class would also work, but I think an extention class is cleaner than a partition class